lib/IDE/Formatting.cpp

```diff
@@ -2485,8 +2485,9 @@ class FormatWalker : public ASTWalker {
   getIndentContextFrom(CaptureListExpr *CL,
                        SourceLoc ContextLoc = SourceLoc()) {
     AbstractClosureExpr *CE = CL->getClosureBody();
+    assert(CE);
```

Comment:
Not here, in the `CaptureListExpr` constructor; sorry if I wasn’t clear.

Reply:
Sorry, I get lost. By constructor do you mean `CaptureListExpr::create`? If so what should I assert there? `assert(mem);`?

Reply:
> By constructor do you mean `CaptureListExpr::create`?

Yes, but preferably the actual constructor, `CaptureListExpr::CaptureListExpr`. It’s defined in the header.

> If so what should I assert there?

`assert(closureBody)`.